styles.css

```diff
@@ -6,20 +6,39 @@ body {
   scroll-behavior: smooth;
 }
 
-.navigation {
```

Comment:
- [ ] Some alterations to previous CSS rules happened and they broke your layout, kindly fix them.

The layout on your master branch ✅:
![image](https://user-images.githubusercontent.com/73140461/146833656-5b8c8634-14e9-476b-ab31-49bdf85a88e3.png)

The layout on your menu-js branch ❌:
![image](https://user-images.githubusercontent.com/73140461/146833715-b1801d96-7ec3-48d8-867f-6182cd2b14bc.png)